### DISCUSSION TITLE
Reference data life

### DISCUSSION THREAD

**Discussion by mwapst** [ivoa/ProposalDM#6] [Ideas]
{Person --- Organisation} ...are we capturing that a person can move org, maybe even belong to more than one.  (Do we need a type of org, eg University, Observatory, Laboratory, etc)

We might want to consider adding start and stop dates to reference data, and particularly state, anyway, eg people belong to an org, people belong to a TAC, availability of observatory equiment, etc so that we don’t lose/ invalidate historical information and eg relating to a person belonging to one org at one time, then a different one at another. Also so that ref data can be excluded from selection once the item is no longer valid.

**Comment by pahjbo** [comment]
> {Person --- Organisation} ...are we capturing that a person can move org, maybe even belong to more than one. (Do we need a type of org, eg University, Observatory, Laboratory, etc)

we are not trying to track that sort of thing in the model itself - the model just provides a "snapshot" of what was true at the time.
> 
> We might want to consider adding start and stop dates to reference data, and particularly state, anyway, eg people belong to an org, people belong to a TAC, availability of observatory equiment, etc so that we don’t lose/ invalidate historical information and eg relating to a person belonging to one org at one time, then a different one at another. Also so that ref data can be excluded from selection once the item is no longer valid.

for tracking people, we are allowing a field for a "long lived" identifier such as https://orcid.org to be used

for observatories and instruments there is the idea of factoring this #4 out of the model entirely to be managed as a separate type of service - however, in order to make progress for the first version of this data model we are keeping them in for now.
